portals/publisher/source/src/app/components/Apis/Details/NewOverview/Endpoints.jsx

```diff
@@ -59,69 +66,173 @@ function Endpoints(props) {
+                                >
+                                    <Typography component='p' variant='body1' className={parentClasses.url}>
+                                        {showEndpoint(api, 'sand')
+                                        && <>{showEndpoint(api, 'sand')}</>}
+                                    </Typography>
+                                </Tooltip>
+                                <Typography component='p' variant='body1' className={parentClasses.notConfigured}>
+                                    {!showEndpoint(api, 'sand') && (
+                                        <>
+                                            <FormattedMessage
+                                                id='Apis.Details.NewOverview.Endpoints.sandbox.not.set'
+                                                defaultMessage='-'
+                                            />
+                                        </>
+                                    )}
+                                </Typography>
+                            </Grid>
+                        )}
                         <Grid item xs={12} md={6} lg={4}>
                             {/* Sandbox Endpoint (TODO) fix the endpoint info when
                                                 it's available with the api object */}
                             <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
                                 <FormattedMessage
-                                    id='Apis.Details.NewOverview.Endpoints.sandbox.endpoint'
-                                    defaultMessage='Sandbox Endpoint'
+                                    id='Apis.Details.NewOverview.Endpoints.endpoint.security'
+                                    defaultMessage='Endpoint Security'
+                                />
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={12} md={6} lg={8}>
+                            <Typography component='p' variant='body1'>
+                                {api.endpointSecurity && <>{api.endpointSecurity.type}</>}
+                            </Typography>
+                            <Typography component='p' variant='body1' className={parentClasses.notConfigured}>
+                                {!api.endpointSecurity
+                                && (
+                                    <>
+                                        <FormattedMessage
+                                            id='Apis.Details.NewOverview.Endpoints.security.not.set'
+                                            defaultMessage='-'
+                                        />
+                                    </>
+                                )}
+                            </Typography>
+                        </Grid>
+                    </Grid>
+                )}
+                { api.advertiseInfo.advertised && (
```

Comment:
use a null check for advertiseInfo

Reply:
Added with https://github.com/wso2/apim-apps/pull/130/commits/e73f69c14b5994afe7b2aa31142b5d0c2dc566e9